Major rework of ecosystem pages (#3060)
open-telemetry/opentelemetry.io: content/en/ecosystem/adopters.md

# All spelling entries must be on a single line
---

OpenTelemetry's mission is to to enable effective observability for all its

> end-users. If you thinking about adopting OpenTelemetry for your organization,

Reviewer: ```suggestion
end-users. If you are thinking about adopting OpenTelemetry for your organization,
```

> you may be curious how other adoption journeys looked like. On this page you can
> find a non-exhaustive list of

Reviewer: ```suggestion
you may be curious about other adoption journeys. The table below is a non-exhaustive list of
```

[end-user organizations](https://community.cncf.io/end-user-community/) that
have adopted OpenTelemetry for
[Observability](/docs/concepts/observability-primer/).

{{% ecosystem/adopters-table %}}

> To have your organization included,
> [add an entry to the list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/adopters.yaml)
> and submit a PR.
> 
> You will need to include[^existing-entries]:

Reviewer: ```suggestion
To have your organization listed, submit a PR with an
entry added to the [adopters list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/adopters.yaml).
The entry must include the following[^grace-period-2024-01-01]:
```

> - a link to a blog post that describes how your organization makes use of
>   OpenTelemetry.
> - a point of contact (GitHub handle or email address), we can reach out for
>   future inquiries.

Reviewer: ```suggestion
- Link to a blog post that describes how your organization makes use of
  OpenTelemetry
- GitHub handle or email address as a point of contact so that we can reach out in case we have questions
```

> Note, that this list is for organizations that adopt OpenTelemetry for
> Observability as an end user and do not provide any kind of services around
> OpenTelemetry themselves.

Reviewer: ```suggestion
Note that this list is for [_end-user_ organizations](https://community.cncf.io/end-user-community/) that  do not provide any OpenTelemetry-related services.
```

> If you provide a library or service made observable through OpenTelemetry, see
> [Integrators](/ecosystem/integrations). <br /> If you provide a solution that

Reviewer: ```suggestion
If your organization provides a **library** or **service** made observable through OpenTelemetry, see
[Integrators](/ecosystem/integrations/).

If your organization provides a solution that
```

Reviewer: The `/ecosystem/integrations/` page is entitled "Integrations", not "Integrators". Either update the title here or rename and retitle the page.

> consumes OpenTelemetry to offer Observability to end users, see

Reviewer: ```suggestion
consumes OpenTelemetry to offer **Observability to end users**, see
```

[Vendors](/ecosystem/vendors).

> [^existing-entries]:

Reviewer: ```suggestion
[^grace-period-2024-01-01]:
```

>     Organizations that have been part of this list before July 2023 are exempted
>     from these requirements until January, 1th 2024.

Reviewer: ```suggestion
    Organizations listed before July 2023 are exempt
    from these requirements until January 1, 2024, after which they will be removed unless updated.
```
